src/styles/blocks/header.scss

```diff
@@ -0,0 +1,69 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
```

Comment:
Consider using a variable for the padding value if it's a common spacing value used in multiple places in the project, for consistency and easier maintenance. If `50px` is a unique value for the header only, then it's fine as is.